File: Iniciante/Beecrowd1012.py
line = input()
line = line.split(" ")
A = float(line[0])
B = float(line[1])
C = float(line[2])
pi = 3.14159

tri = A*C/2
cir = pi*C**2
trap = (A+B)*C/2
qua = B**2
ret = A*B
print(f"TRIANGULO: {tri:.3f}")
print(f"CIRCULO: {cir:.3f}")
print(f"TRAPEZIO: {trap:.3f}")
print(f"QUADRADO: {qua:.3f}")
print(f"RETANGULO: {ret:.3f}")

File: Iniciante/C++/Beecrowd1012.cpp
#include <iostream>
#include <iomanip>

#define PI 3.14159

using namespace std;

int main (void) {
    double A, B, C, area;
    
    cin >> A >> B >> C;
    
    cout << fixed << setprecision(3);

    area = (A*C)/2.0;
    cout << "TRIANGULO: " << area << endl;
    area = (PI*C*C);
    cout << "CIRCULO: " << area << endl;
    area = (A + B)*C/2.0;
    cout << "TRAPEZIO: " << area << endl;
    area = B*B;
    cout << "QUADRADO: " << area << endl;
    area = A * B;
    cout << "RETANGULO: " << area << endl;      
    
    return 0;
}